.github/workflows/startup-perf-ab.yml

```
@@ -0,0 +1,190 @@
+  contents: read
+
+jobs:
+  startup-perf-ab:
+    name: interleaved startup A/B
+    runs-on: ubuntu-latest
+    timeout-minutes: 90
+    env:
+      NODE_OPTIONS: --max-old-space-size=4096
+      # Outside the repo: each arm is a full worktree with its own node_modules,
+      # not a stray untracked directory inside the checkout.
+      ARMS_DIR: ${{ runner.temp }}/orca-bench-arms
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)